manifests/plugin/python.pp

```diff
@@ -20,26 +15,39 @@
   }
   # End deprecation
 
-  file {
-    "${name}.load":
-      ensure  => $ensure,
-      path    => "${conf_dir}/${order}-${name}.conf",
-      owner   => 'root',
-      group   => $collectd::params::root_group,
-      mode    => '0640',
-      content => template('collectd/python.conf.erb'),
-      notify  => Service['collectd'],
+  $ensure_modulepath = $ensure ? {
+      'absent' => $ensure,
+      default  => 'directory',
   }
 
-  file {
-    "${name}.script":
-      ensure  => $ensure,
-      path    => "${modulepath}/${module}.py",
-      owner   => 'root',
+  file { $modulepath :
+      ensure  => $ensure_modulepath,
+      mode    => '0750',
+      owner   => root,
       group   => $collectd::params::root_group,
-      mode    => '0640',
-      source  => $script_source,
-      require => File["${name}.load"],
-      notify  => Service['collectd'],
+      require => Package[$collectd::params::package]
+  }
+
+  collectd::plugin { 'python' :
+      ensure   => $ensure,
+      interval => $interval,
```

Comment:
From the LoadPlugin doc : "[interval] Sets a plugin-specific interval for collecting metrics. This overrides the global Interval setting. If a plugin provides own support for specifying an interval, that setting will take precedence."

I am not familiar with Python nor the Python plugin of collectd ; http://collectd.org/documentation/manpages/collectd-python.5.shtml Are the notion of interval in the python plugin mandatory or could they inherit the value from collectd ?